internal/config/local.go

```diff
@@ -4,6 +4,7 @@ type (
 	Config struct {
 		Password         string            `yaml:"password"`
 		WifiConfigs      []WifiConfig      `yaml:"wifiConfigs"`
+		EthernetConfigs  []EthernetConfig  `yaml:"wiredConfig"`
```

Comment:
This shouldn't be an array, there is only ever one EthernetConfig. The yaml file will need to be updated as well.